Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -345,6 +346,9 @@ def bump_formula_pr
       alias_rename.map! { |a| formula.tap.alias_dir/a }
     end
 
+    ohai "brew bump-python-resources #{formula.name}"
```

Comment:
I see you are using interchangeably bump-python-resources and update-python-resources. Dunno which one is correct :)

Reply:
Good catch, thanks. It should be update-python-resources (although I'm open to changing the name is desired).

I'll make sure fix these